Dockerfile

```
@@ -0,0 +1,29 @@
+FROM node:14.17.4 AS restore
+WORKDIR /src
+COPY package.json yarn.lock ./
+RUN yarn
+COPY . .
+
+FROM restore AS verify-format
+RUN yarn verify-format
+
+FROM koalaman/shellcheck-alpine as verify-sh
+WORKDIR /src
+COPY ./*.sh ./
+RUN shellcheck -e SC1091,SC1090 ./*.sh
+
+FROM restore as build
+COPY . .
+ARG baseUrl=https//cdn.fromdoppler.com
+ARG pkgName=unlayer-editor
+ARG version=v0.0.0-build0
+RUN yarn build --cdn "${baseUrl}/${pkgName}/${version}"
```

```suggestion
RUN yarn build
```

Creo que see parámetro aún no lo usamos, no?

---

se podría quitar?

---

Changed.